torch/_inductor/select_algorithm.py

```diff
@@ -197,6 +197,11 @@ def store_output(self, indices, val, mask):
                 V.graph.sizevars.simplify(s) for s in self.output_node.get_size()
             ]
             assert len(indices) == len(lengths)
+            replacements = {
+                x: self.args.size(x)
+                for x in index_symbols + lengths
+                if isinstance(x, sympy.Symbol) and is_size_var(x.name)
+            }
```

Comment:
I think self.rename_indexing() might do the same thing as this.

Reply:
Ah you're right. I thought rename_indexing() was only for single symbol replacement previously.